tempesta_fw/cache.c

```diff
@@ -151,10 +151,12 @@ static const int hbh_hdrs[] = {
 	[TFW_HTTP_HDR_CONNECTION]	= 1,
 };
 
-static struct {
-	int	cpu;
-	TDB	*db;
-} c_nodes[MAX_NUMNODES] __read_mostly;
+static struct ca_node_t {
+	int		cpu[NR_CPUS];
+	unsigned int	cpu_idx;
+	unsigned int	nr_cpus;
+	TDB		*db;
+} ca_nodes[MAX_NUMNODES];
```

Comment:
`ca` is much more readable and understandable than `c` as an abbreviation for `cache`.
`__read_mostly` is not suitable anymore, in my opinion, as `cpu_idx` is continuously incremented now.